node/src/keyshare.rs

```diff
@@ -269,6 +269,19 @@ impl KeyshareStorage {
     }
 }
 
+#[cfg(test)]
+impl KeyshareStorage {
+    // Do NOT use this for production
```

Comment:
Make it a doc instead of comment.
```suggestion
    /// Do NOT use this for production
```